apps/ui/src/views/Space/Settings.vue

```diff
@@ -312,7 +312,7 @@ watchEffect(() => setTitle(`Edit settings - ${props.space.name}`));
   </div>
   <div
     v-else
-    class="space-y-4 pb-8"
+    class="space-y-4 pb-[100px]"
```

Comment:
Any advantages of using 100px instead of pb-8? 🤔

Reply:
pb-8 is not tall enough to fit the bottom toolbar, which is 80px tall